wuzz.go

```diff
@@ -495,6 +496,7 @@ func (a *App) SetKeys(g *gocui.Gui) {
 
 	g.SetKeybinding("", gocui.KeyCtrlR, gocui.ModNone, a.SubmitRequest)
 	g.SetKeybinding("url", gocui.KeyEnter, gocui.ModNone, a.SubmitRequest)
+	g.SetKeybinding("", gocui.KeyCtrlU, gocui.ModNone, a.ExportAsCURLCommand)
```

Comment:
I tested it, and <kbd>ctrl+u</kbd> does not work for me.. =\ It is the "delete line" function in bash/zsh: https://ss64.com/bash/syntax-keyboard.html . What about switching back to <kbd>ctrl+e</kbd> by default? Sorry for the wrong advice..